config/infra-dev/config.yaml

```diff
@@ -1,2 +1,5 @@
 sso_admin_role:
   arn: arn:aws:iam::035458030717:role/aws-reserved/sso.amazonaws.com/AWSReservedSSO_Administrator_580e9f32ac55c4e7
+budgets:
+  compute: 1000
```

Comment:
Just as a reminder for myself, these budgets override what's in `config/config.yaml` correct?

Reply:
Yup! https://docs.sceptre-project.org/dev/docs/stack_group_config.html#stack-group-config-cascading-config